lib/client.ts

```diff
@@ -115,8 +115,19 @@ export class Client {
       () => notNil(params.id),
       collectionUrl => `${collectionUrl}/${params.id}`
     ),
-    (hostname) => `${hostname}${S(base).template(params, '{', '}').s}`
+    (hostname) => `${hostname}/${this.version(api_version)}${S(base).template(params, '{', '}').s}`
   )(this.host)
+
+  private version = (api_version: string): string => {
```

Comment:
Line 122 can probably be removed
```suggestion
  private version = (api_version: string = 'v1.0'): string => {
```

Unless there is a valid case when `api_version` can equal `''`, in which case the following should do the job:

```
const finalizedApiVersion = apiVersion || 'v1.0'
```